lms/djangoapps/commerce/api/v1/serializers.py

```diff
@@ -17,19 +20,60 @@ def get_identity(self, data):
         except AttributeError:
             return None
 
+    def get_verification_deadline(self, obj):
+        """ Return the course mode's verification deadline.
+
+        Returns None, if the mode is not a verified mode.
+        """
+        if CourseMode.is_verified_mode(obj.to_tuple()):
+            return VerificationDeadline.deadline_for_course(obj.course_id)
+
+        return None
+
     class Meta(object):  # pylint: disable=missing-docstring
         model = CourseMode
-        fields = ('name', 'currency', 'price', 'sku', 'expires')
+        fields = ('name', 'currency', 'price', 'sku', 'expires', 'verification_deadline')
 
 
 class CourseSerializer(serializers.Serializer):
     """ Course serializer. """
+    # Note (CCB): This is a temporary placeholder. It is simpler to hold this data on the serializer than
+    # find a way to update the model(s) to do it for us.
+    verification_deadline = None
+
     id = serializers.CharField()  # pylint: disable=invalid-name
+    name = serializers.CharField(read_only=True)
     modes = CourseModeSerializer(many=True, allow_add_remove=True)
 
+    def __init__(self, *args, **kwargs):
+        if kwargs.get('data', None):
+            candidate = None
+
+            for mode in kwargs['data'].get('modes', []):
+                if CourseMode.is_verified_slug(mode['name']):
+                    candidate = mode.get('verification_deadline', None)
```

Comment:
With the advent of the VerificationDeadline table, verification deadlines are now simply associated with a course ID. That being the case, I think we'll want to treat the verification deadline like the course name (i.e., top-level property of the course, not nested within the modes/products).